t/rest_server/.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'deploy'
+
+on:
+  workflow_dispatch:
+    inputs:
+      deploymentEnvironment:
```

Comment:
The style for github workflow files seems to avoid snakeCase. words_with_underscores looks like the style. https://github.com/search?q=org%3Aabcxyz+workflow_call&type=code

Reply:
Changed to snake_case